test/component/testDocumentsPage.js

```diff
@@ -1190,6 +1190,7 @@ describe('documents', () => {
                         'old_sendDocumentsAddress',
                         'text6',
                         'checklist1-item1-codicils',
+                        'checklist2-item3-will-uploaded',
```

Comment:
This test is testing that the content is not displayed if the user has uploaded a document. 

We also need a similar test to check that the content is displayed if the user has not uploaded a document...uploading a document is not a requirement.